### DISCUSSION TITLE
Game Won't Launch

### DISCUSSION THREAD

**Discussion by JakeHRobinson** [ppy/osu#15530] [Q&A] [answered]
I logged in to play today and when I got on, it said there was an update. I've updated and now the game won't launch. I double click the .exe and quite literally nothing happens, it doesn't even try to run. The updater (in the lazer folder) will open for 0.1 of a second and then closes again. 

This is the issue I get, If I've linked the wrong thing do let me know, I've never posted an issue before so I apologise if the formatting is incorrect. 

![image](https://user-images.githubusercontent.com/69173248/140835554-161451dd-4257-4db2-9ada-9e2ca526c557.png)

**Comment by nahkd123** [comment]
Are there any logs? If so please attach them here. See https://github.com/ppy/osu/discussions/15188 for information on finding and attaching logs

**Comment by JakeHRobinson** [comment]
These are all the logs in the file path it said to go to, not sure if these are standard or lazer (or if they all get sent to the same place). 



[performance.log](https://github.com/ppy/osu/files/7501053/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7501054/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7501055/updater.log)
[database.log](https://github.com/ppy/osu/files/7501056/database.log)
[network.log](https://github.com/ppy/osu/files/7501057/network.log)

**Comment by peppy** [comment]
Please delete `client.realm` from your osu! folder. It should solve the issue. This is an actual issue at our end while I'll look to fix, but you shouldn't lose anything other than your key bindings by deleting that file to resolve things in the short term.

**Reply by JakeHRobinson** [reply]
This is most likely me being dumb, however, I don't appear to have that in either my osu! folder or my osu!lazer folder 

![image](https://user-images.githubusercontent.com/69173248/140845004-b27fe464-0a41-4995-bad2-ab952a1bb967.png)
![image](https://user-images.githubusercontent.com/69173248/140845033-bd04cd2f-48d7-46f9-94c8-8f1ed4c4c703.png)

**Reply by peppy** [reply]
It's where you found the logs. You don't need to search it should be there in plain view.

**Reply by JakeHRobinson** [reply]
![image](https://user-images.githubusercontent.com/69173248/140855042-3191aded-cead-434f-ac38-ba96e5bb6ebe.png)

**Reply by JakeHRobinson** [reply]
![image](https://user-images.githubusercontent.com/69173248/140855116-4f26c6c0-f9b5-471a-b68b-9149a4f40775.png)

**Reply by frenzibyte** [reply]
The parent folder of the `logs` folder in the first picture, or simply `%AppData%/osu/client.realm`.

For reference, the parent folder of the second image belongs to osu!(stable), which is out-of-scope from here.

**Reply by peppy** [reply]
`Roaming/osu!/client.realm`

**Comment by peppy** [comment, marked as answer]
Will be resolved by https://github.com/ppy/osu/pull/15533. @Phoenix21938 if you still can't find the file, please wait for the next release and install it over the top of your existing one.

**Reply by JakeHRobinson** [reply]
After deleting that file and then reinstalling the game, it's all working perfectly. Thank you for the help! :)
